python/cudf/cudf/tests/test_orc.py

```diff
@@ -795,3 +795,22 @@ def test_orc_string_stream_offset_issue():
     df.to_orc(buffer)
 
     assert_eq(df, cudf.read_orc(buffer))
+
+
+def test_orc_reader_decimal(datadir):
+    path = datadir / "TestOrcFile.decimal.orc"
+    try:
+        orcfile = pa.orc.ORCFile(path)
+    except pa.ArrowIOError as e:
+        pytest.skip(".orc file is not found: %s" % e)
+
+    pdf = orcfile.read().to_pandas()
+    gdf = cudf.read_orc(
+        path, engine="cudf", decimal_cols_as_float=["_col0"]
```

Comment:
I think we should also test with a column name that's not in the table, to verify that we're actually matching the column name(s). 
Also should have a test with `decimal_cols_as_float=["fake col name", "_col0", "another fake name"]` to verify that we're not just matching the first name in the list.

Reply:
Good idea